test/e2e/operator_test.go

```diff
@@ -302,8 +311,10 @@ func TestDNSForwarding(t *testing.T) {
 	}()
 
 	// Wait for the upstream resolver Pod to be ready.
+	name := types.NamespacedName{Namespace: upstreamResolver.Namespace, Name: upstreamResolver.Name}
 	err = wait.PollImmediate(1*time.Second, 2*time.Minute, func() (bool, error) {
-		if err := cl.Get(context.TODO(), types.NamespacedName{Namespace: upstreamResolver.Namespace, Name: upstreamResolver.Name}, upstreamResolver); err != nil {
+		if err := cl.Get(context.TODO(), name, upstreamResolver); err != nil {
+			t.Logf("failed to get pod %s: %v", name, err)
```

Comment:
The log msg will look better with:
`t.Logf("failed to get deployment %s/%s: %v", name.Namespace, name.Name, err)`